lib/types/index.js

```diff
@@ -593,11 +593,21 @@ type Frequency = {
 }
 
 export type Trip = {|
+  bikes_allowed: ?boolean,
+  blockId: string,
+  directionId: number,
   frequencies: Array<Frequency>,
   id: null | number,
+  pattern_id: string,
+  route_id: string,
+  service_id: string,
+  shape_id: string,
   stopTimes: Array<StopTime>,
+  tripHeadsign: string,
   tripId: string,
-  useFrequency: boolean
+  tripShortName: ?string,
+  useFrequency?: boolean,
```

Comment:
I believe if the value is undefined in the database, it will not come through in the response of the GraphQL query. If you look at a response from GraphQL in chrome dev tools you should see this.